web/angular.json

```diff
@@ -57,10 +57,7 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "taskana-web:build",
-            "sourceMap": {
-              "scripts": true
-            }
```

Comment:
why did you remove the sourceMap?

Reply:
this usage is deprecated
but a complete removal was unnecessary 
i will readd it